components/locale/src/fallback/mod.rs

```diff
@@ -108,16 +108,12 @@ impl LocaleFallbacker {
     #[cfg(feature = "compiled_data")]
     #[allow(clippy::new_ret_no_self)] // keeping constructors together
     pub const fn new<'a>() -> LocaleFallbackerBorrowed<'a> {
```

Comment:
Question: why aren't we just returning 'static?

Reply:
Because the compiler cannot downcast this lifetime, and compiled data needed that somwhere.